app/controllers/api/v2/compute_attributes_controller.rb

```diff
@@ -22,8 +22,8 @@ class ComputeAttributesController < V2::BaseController
 
       def create
         @compute_attribute = ComputeAttribute.new(compute_attribute_params.merge(
-          :compute_profile_id => params[:compute_profile_id],
-          :compute_resource_id => params[:compute_resource_id]))
+                                                    :compute_profile_id => params[:compute_profile_id],
```

Comment:
I don't like this too much but I don't insist on disabling, I think whitespace formatting causes most cop failures and leads to frustrations like "why my PR is red now?"

Reply:
I do understand that, on the other hand it's an auto-correctable error and editors can be configured for it, too... This change did only touch eleven files of the whole codebase, so I guess most of the code is already like wanted by the cop.

However, I'd like to vote here with 👍 and 👎 for this cop.

Reply:
If I'm the only one, feel free to skip voting, 
not a big deal :-)

Reply:
I prefer the original formatting.

Reply:
How about removing this one commit from this PR and I'll open another one (or a gist) with the variants possible?

I think that the "consistent" option of this cop might fancy, what you prefer while still enforcing the same alignment of all concerning lines.

Reply:
@lzap please have a look again. 👀

Reply:
> How about removing this one commit from this PR and I'll open another one (or a gist) with the variants possible? 

Sounds good to me